### ISSUE TITLE
Gitlens not using global config settings

### ISSUE THREAD

**Issue by sfuerte** [eamodio/vscode-gitlens#758]
- GitLens Version: 9.8.1
- VSCode Version: 1.34.0 
- OS Version: macOS 10.14.5 (18F203)

My git settings:
```
$ git config --global -l
user.name=<full name>
user.email=<corp email>
```

Whether it's related to https://github.com/Microsoft/vscode/issues/67736 or not, but Gitlens not correctly using my global user.name and user.email. Initially thought that the problem was with VSCode, but a quick lookup to a raw commit in Bitbucket shows that it's fine there:
```
From 3643...
From: <full name> <corp email>
Date: Thu, 30 May 2019 23:08:20 -0600
...
```

At the same time, Gitlens showing the following under Contributors:
- <full name> <corp email> (the correct entry)
- <account's name> <"account name"@"localhost name"> (the incorrect entry)

**Comment by eamodio** [comment]
@sfuerte GitLens only uses the name/email for the current user for determining "You", other than that it just displays the information stored with each commit. The only caveat to that is GitLens also supports git's .mailmap file which allows for changing (merging) multiple names/emails into a single canonical version.

Can you provide more concrete details on what you are seeing and where you think the problem lies? Thanks!

**Comment by sfuerte** [comment]
@eamodio, emailed you directly due to privacy.

**Comment by eamodio** [comment]
@sfuerte I've just published v9.8.2 which fixes an issue with the Contributors view -- can you see if it resolves the issue for you?

**Comment by sfuerte** [comment]
@eamodio, yes, it does. Thanks a lot!
